.gitignore

```diff
@@ -39,3 +39,4 @@ bower.json
 
 # Ignore Byebug command history file.
 .byebug_history
+/config/database.yml
```

Comment:
i went with the `.example` approach but `.env` sounds reasonable to me if you want to set it up